.travis.yml

```diff
@@ -1,18 +1,20 @@
-language: python
-before_install:
-  - sudo apt-get update
-
-  # First install libcangjie
-  - sudo apt-get install libsqlite3-dev
-  - git clone git://github.com/Cangjians/libcangjie
-  - cd libcangjie && git checkout 1.x && ./autogen.sh --prefix=/usr && make && sudo make install && cd ..
-  - git clean -dfx
-
-  # Next install pycangjie
-  - sudo apt-get install cython python3-dev
-  - git clone git://github.com/Cangjians/pycangjie
-  - cd pycangjie && git checkout 1.x && ./autogen.sh --prefix=/usr && make && sudo make install && cd ..
-  - git clean -dfx
-
-  - sudo apt-get install autopoint intltool libibus-1.0-dev python3-gobject
-script: ./autogen.sh && make && make check && make distcheck
+language: c
+sudo: required
+branches:
+  only:
+    - master
+services:
+  - docker
```

Comment:
If you move the `services` section just after the `env` one, then all our `.travis.yml` files are very similar, making them faster to skim through and compare.

I won't block the review on this though, because it's trivial and not very important, but it would be nice. 🙂